CHANGES

```diff
@@ -68,6 +81,9 @@ Standard Library
   and, consequently, choice of representatives in equivalence classes.
   Various proof-theoretic characterizations of choice over setoids in
   file ChoiceFacts.v.
+- New lemmas about iff and about orders on positive and Z.
+- New lemmas on powerRZ.
+- Strengthened statement of JMeq_eq_dep (closes BZ#4912).
 - The BigN, BigZ, BigZ libraries are not part anymore of Coq standard
```

Comment:
To me, that phrase sounds a bit awkward.

How about "are no longer part of the Coq standard ..." ?